service-commands/scripts/utils.sh

```diff
@@ -38,8 +38,19 @@ copy_file_to_remote() {
 	remote_file=$5
 
 	case "$provider" in
-		azure) exit 1 ;;  # TODO!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-		gcp) eval "gcloud compute scp $local_file $user@$name:$remote_file" ;;
+		azure)
+			# old logic that was buggy for gcloud
+			cat $local_file | eval "$(get_ssh_args $provider $user $name)" "cat > $remote_file"
+
+			# new untested logic for Azure:
```

Comment:
Let's include the logic after we've tested it? I'm not the biggest fan of merging in uncommented and untested code in when we can leave it resting in an open branch or draft PR. Just been bit a few times by this 😓